packages/visual-editor/src/editor/YextStructFieldSelector.tsx

```diff
@@ -31,8 +31,8 @@ export type SelectorPropsV2 = {
   filter: EntityFieldTypesFilter;
 };
 
-// YextSelectorV2 will be used for new built-in struct and list field types.
-export const YextSelectorV2 = <U extends Record<string, any>>(
+// YextStructFieldSelector will be used for new built-in struct and list field types.
```

Comment:
Rename YextEntityFieldV2 and any other new things you added that are named like "V2"